packages/grafana-ui/src/components/index.ts

```diff
@@ -9,12 +9,16 @@ export { IndicatorsContainer } from './Select/IndicatorsContainer';
 export { NoOptionsMessage } from './Select/NoOptionsMessage';
 export { default as resetSelectStyles } from './Select/resetSelectStyles';
 
+// Forms
+export { GfFormLabel } from './GfFormLabel/GfFormLabel';
```

Comment:
Can we get rid of this in favour of `Label`? :)

Reply:
yes, if we rename it FormLabel or something, since we only have one namespace for @grafana/ui we need to have a bit more specific naming